### ISSUE TITLE
Error "phonegap"

### ISSUE THREAD

**Issue by Narjous** [npm/npm#8065]
des ereurs declenchés lors l'installation de phonegap, dans ma commande je fais : 
```
npm -g install phonegap ,il m'arrive:npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```
c quoi la solution !

**Comment by KenanY** [comment]
@Narjous What's the error? Node.js version? `npm` version? System information? If a `npm-debug.log` is generated, could you upload it as a [gist](https://gist.github.com/)? https://github.com/npm/npm/wiki/Contributing-Guidelines#a-great-bug-report-contains

**Comment by Narjous** [comment]
j'ai installé node.js,,mai je n'arrive pas à installer phonegap
ce que est affiché en haut il m'arrive ,je sai pas comment faire :(
ceca ma commande en fait :
Microsoft Windows [version 6.1.7600]
Copyright (c) 2009 Microsoft Corporation. Tous droits réservés.

C:\Users\ATIA>npm -g install phonegap
npm ERR! Windows_NT 6.1.7600
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs
\\node_modules\\npm\\bin\\npm-cli.js" "-g" "install" "phonegap"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4

npm ERR! shasum check failed for C:\Users\ATIA\AppData\Local\Temp\npm-5992-876e3
84e\registry.npmjs.org\phonegap-build\-\phonegap-build-0.9.1.tgz
npm ERR! Expected: a91df8a3b6e8e860aa707cc33b15bc6a38139acc
npm ERR! Actual:   18ce57ff826702a64e5f8ac821bd36580a0f7cfa
npm ERR! From:     https://registry.npmjs.org/phonegap-build/-/phonegap-build-0.
9.1.tgz
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!    
 C:\Users\ATIA\npm-debug.log

**Comment by KenanY** [comment]
@Narjous Please try:

``` sh
$ npm cache clean phonegap-build
```

**Comment by Narjous** [comment]
oki,merci ;)

**Comment by ehsalazar** [comment]
It sounds like you got things worked out. I'll close this for now as we’re trying to clean up some old issues. If not, please reply and let us know. We’ll be happy to reopen if necessary.

**Comment by Narjous** [comment]
okay as u like,,i've finished with phonegap :) ;)

2015-08-12 5:31 GMT+02:00 Ernie Salazar <notifications@github.com>:

> Closed #8065 <https://github.com/npm/npm/issues/8065>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/npm/npm/issues/8065#event-379890591>.
>



-- 
Cordialement,
Narjes Atia
